src/libraries/tick_bitmap.cairo

```diff
@@ -205,4 +124,60 @@ mod TickBitmap {
             }
         }
     }
+
+    fn generate_hashed_word_pos(word_pos: @i16) -> felt252 {
```

Comment:
I removed this function because adding the `Hash` tag to the struct allows us to generate a hash in a simpler way
```
#[derive(Hash)]
struct StructForHash {
    first: (),
    second: felt252,
    third: (felt252, felt252),
}
assert_eq(
        @PoseidonTrait::new()
            .update_with(StructForHash { first: (), second: 10, third: (6, 17) })
            .finalize(),
        @PoseidonTrait::new().update(10).update(6).update(17).finalize(),
        'Bad hash of StructForHash',
    );
```
[source](https://github.com/starkware-libs/cairo/blob/04c2ce64b487b0f459ae9152452e04260e726c7c/corelib/src/test/hash_test.cairo#L83)